docs/UserGuide.md

```diff
@@ -117,14 +116,28 @@ Format: `edit INDEX [n/NAME] [p/PHONE] [e/EMAIL] [a/ADDRESS] [t/TAG]…​`
     specifying any tags after it.
 
 Examples:
-*  `edit 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
-*  `edit 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+*  `editcontact 1 p/91234567 e/johndoe@example.com` Edits the phone number and email address of the 1st person to be `91234567` and `johndoe@example.com` respectively.
+*  `editcontact 2 n/Betsy Crower t/` Edits the name of the 2nd person to be `Betsy Crower` and clears all existing tags.
+
+### Deleting a person : `delete`
 
-### Locating persons by name: `find`
+Deletes the specified person from the address book.
+
+Format: `deletecontact INDEX`
```

Comment:
do we use the INDEX prefix, or no?
I feel it's better if we do

Reply:
for INDEX input, do we use an "i/" prefix?
To maintain consistency with other commands that we have implemented.

Reply:
Alright, as long as we're on the same page